[2025-01-22T00:01:26.635+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-22T00:01:26.704+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: detect_data_drift_notify_retrain.prepare_email_no_drift_task manual__2025-01-21T23:49:48.622951+00:00 [queued]>
[2025-01-22T00:01:26.746+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: detect_data_drift_notify_retrain.prepare_email_no_drift_task manual__2025-01-21T23:49:48.622951+00:00 [queued]>
[2025-01-22T00:01:26.748+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-01-22T00:01:26.818+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): prepare_email_no_drift_task> on 2025-01-21 23:49:48.622951+00:00
[2025-01-22T00:01:26.858+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=328) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-22T00:01:26.860+0000] {standard_task_runner.py:72} INFO - Started process 333 to run task
[2025-01-22T00:01:26.900+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'detect_data_drift_notify_retrain', 'prepare_email_no_drift_task', 'manual__2025-01-21T23:49:48.622951+00:00', '--job-id', '231', '--raw', '--subdir', 'DAGS_FOLDER/monitoring-dag.py', '--cfg-path', '/tmp/tmp9f_7_ozj']
[2025-01-22T00:01:26.911+0000] {standard_task_runner.py:105} INFO - Job 231: Subtask prepare_email_no_drift_task
[2025-01-22T00:01:27.222+0000] {task_command.py:467} INFO - Running <TaskInstance: detect_data_drift_notify_retrain.prepare_email_no_drift_task manual__2025-01-21T23:49:48.622951+00:00 [running]> on host d3de575648b4
[2025-01-22T00:01:27.603+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='RL' AIRFLOW_CTX_DAG_ID='detect_data_drift_notify_retrain' AIRFLOW_CTX_TASK_ID='prepare_email_no_drift_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-21T23:49:48.622951+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-21T23:49:48.622951+00:00'
[2025-01-22T00:01:27.608+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-22T00:01:27.787+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-22T00:01:27.856+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-22T00:01:27.860+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=detect_data_drift_notify_retrain, task_id=prepare_email_no_drift_task, run_id=manual__2025-01-21T23:49:48.622951+00:00, execution_date=20250121T234948, start_date=20250122T000126, end_date=20250122T000127
[2025-01-22T00:01:27.974+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-22T00:01:28.121+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-22T00:01:28.145+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
